gitgud/levels/rampup/_reversing/setup.spec

```diff
@@ -0,0 +1,7 @@
+1 (master)
+2 : 1 (local)
```

Comment:
Change local to master and remove master

Reply:
Changed

Reply:
Can you make sure master is checked out too?

Reply:
done